.circleci/config.yml

```diff
@@ -0,0 +1,45 @@
+version: 2.1
+
+jobs:
+  lint:
+    docker:
+      - image: rust
+    steps:
+      - checkout
+      - run:
+          name: Install cargo fmt
+          command: rustup component add rustfmt
+      - run:
+          name: Run lint
+          command: cargo fmt -- --check
+          
+  build:
+    parameters:
+      toolchain:
+        description: rust toolchain
+        type: string
+
+    docker:
+      - image: buildpack-deps:trusty
+      - image: juxt/crux-standalone:20.07-1.10.0
+
+    steps:
+      - checkout
+      - run: >
+          curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | 
+          sh -s -- -v -y --profile minimal --default-toolchain <<parameters.toolchain>>
+      - run: $HOME/.cargo/bin/cargo build
+      - run: $HOME/.cargo/bin/cargo test --examples
+      - run: $HOME/.cargo/bin/cargo test --examples --features "async"
+      - run: $HOME/.cargo/bin/cargo test --locked  --no-fail-fast --lib
+      - run: $HOME/.cargo/bin/cargo test --test lib --no-fail-fast --features "mock"
+      
+workflows:
+  version: 2.1
+
+  build_and_test:
+    jobs:
```

Comment:
You need to add a `- lint` here just like the `- build` one, but without all the extra info, so it will run

Reply:
dac9a84